docs/submitting-applications.md

```diff
@@ -177,8 +177,9 @@ debugging information by running `spark-submit` with the `--verbose` option.
 
 # Advanced Dependency Management
 When using `spark-submit`, the application jar along with any jars included with the `--jars` option
-will be automatically transferred to the cluster. Spark uses the following URL scheme to allow
-different strategies for disseminating jars:
+will be automatically transferred to the cluster. URLs supplied after --jars must be separated by commas. Each entry points to a specific jar file, resulting in a comma-separated list of local jars. That list is included on the driver and executor classpaths. Directory expansion does not work with --jars. 
```

Comment:
This seems OK but do they have to be local JARs (I think so)? in which case are they really URLs? The second sentence you added seems to say the same thing as the first then. You could back-tick `--jars` too for consistency.